docs/user-guide/namespaces.md

```diff
@@ -196,36 +111,12 @@ is local to a namespace.  This is useful for using the same configuration across
 multiple namespaces such as Development, Staging and Production.  If you want to reach
 across namespaces, you need to use the fully qualified domain name (FQDN).
 
-### REST API
-
-To interact with the Namespace API:
-
-| Action | HTTP Verb | Path | Description |
-| ------ | --------- | ---- | ----------- |
-| CREATE | POST | /api/{version}/namespaces | Create a namespace |
-| LIST | GET | /api/{version}/namespaces | List all namespaces |
-| UPDATE | PUT | /api/{version}/namespaces/{namespace} | Update namespace {namespace} |
-| DELETE | DELETE | /api/{version}/namespaces/{namespace} | Delete namespace {namespace} |
-| FINALIZE | POST | /api/{version}/namespaces/{namespace}/finalize | Finalize namespace {namespace} |
-| WATCH | GET | /api/{version}/watch/namespaces | Watch all namespaces |
-
-To interact with content associated with a Namespace:
-
-| Action | HTTP Verb | Path | Description |
-| ---- | ---- | ---- | ---- |
-| CREATE | POST | /api/{version}/namespaces/{namespace}/{resourceType}/ | Create instance of {resourceType} in namespace {namespace} |
-| GET | GET | /api/{version}/namespaces/{namespace}/{resourceType}/{name} | Get instance of {resourceType} in namespace {namespace} with {name} |
-| UPDATE | PUT | /api/{version}/namespaces/{namespace}/{resourceType}/{name} | Update instance of {resourceType} in namespace {namespace} with {name} |
-| DELETE | DELETE | /api/{version}/namespaces/{namespace}/{resourceType}/{name} | Delete instance of {resourceType} in namespace {namespace} with {name} |
-| LIST | GET | /api/{version}/namespaces/{namespace}/{resourceType} | List instances of {resourceType} in namespace {namespace} |
-| WATCH | GET | /api/{version}/watch/namespaces/{namespace}/{resourceType} | Watch for changes to a {resourceType} in namespace {namespace} |
-| WATCH | GET | /api/{version}/watch/{resourceType} | Watch for changes to a {resourceType} across all namespaces |
-| LIST | GET | /api/{version}/list/{resourceType} | List instances of {resourceType} across all namespaces |
-
-## Design
+## Not All Objects are in a Namespace
 
-Details of the design of namespaces in Kubernetes, including a [detailed example](../design/namespaces.md#example-openshift-origin-managing-a-kubernetes-namespace)
-can be found in the [namespaces design doc](../design/namespaces.md)
+Most kubernetes resources (e.g. pods, services, replication controllers, and others) are
+in a some namespace.  However namespace resources are not themselves in a namespace.
+And, low-level resources, such as [nodes](../../docs/admin/node.md) and
+persistentVolumes, are not in any namespace.
```

Comment:
Add something like: Events are an exception: they may or may not have a namespace, depending on the object the event is about.